### PR TITLE
allow config file for brakeman

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: lib/polish_geeks/dev_tools/commands/base.rb [hunk 1]
```diff
@@ -7,6 +7,8 @@ module Commands
       class Base
         # Output string that we get after executing this command
         attr_reader :output
+        # Config manager for command
```
Comment (mensfeld) [inline lib/polish_geeks/dev_tools/commands/base.rb]: could you explain in more details what for/how is it going to be used?

File: lib/polish_geeks/dev_tools/config_manager.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
+module PolishGeeks
+  module DevTools
+    # Config manager for commands
+    class ConfigManager
+      def initialize(file_name)
```
Comment (mensfeld) [inline lib/polish_geeks/dev_tools/config_manager.rb]: readme missing + examples

File: lib/polish_geeks/dev_tools/config_manager.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        @file_name = file_name
+      end
+
+      # @return [Boolean] if there is a config present
+      def present?
```
Comment (mensfeld) [inline lib/polish_geeks/dev_tools/config_manager.rb]: example missing

File: lib/polish_geeks/dev_tools/config_manager.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        application? || local?
+      end
+
+      # @return [String] path to config file
+      def path
+        application? ? application_path : local_path
```
Comment (mensfeld) [inline lib/polish_geeks/dev_tools/config_manager.rb]: example missing, missing what it does

File: lib/polish_geeks/dev_tools/config_manager.rb [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      end
+
+      private
+
+      # @return [Boolean] if there is a config present locally
+      def local?
```
Comment (mensfeld) [inline lib/polish_geeks/dev_tools/config_manager.rb]: example missing, missing what it does

File: lib/polish_geeks/dev_tools/config_manager.rb [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        !@file_name.nil? && !local_path.nil?
+      end
+
+      # @return [String] path to local config file
+      def local_path
```
Comment (mensfeld) [inline lib/polish_geeks/dev_tools/config_manager.rb]: example missing

File: lib/polish_geeks/dev_tools/config_manager.rb [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        fetch_path(DevTools.gem_root)
+      end
+
+      # @return [Boolean] if there is a config present in the application
+      def application?
+        !@file_name.nil? && !application_path.nil?
```
Comment (mensfeld) [inline lib/polish_geeks/dev_tools/config_manager.rb]: !@file_name.nil? && !application_path.nil? isnt that the same as @file_name && @application_path ?

File: lib/polish_geeks/dev_tools/config_manager.rb [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+      end
+
+      # @return [String] path to application config file
+      def application_path
+        fetch_path(DevTools.app_root)
+      end
+
+      # @return [String] path to config file if it exists
+      def fetch_path(start_path)
+        ['', 'config'].each do |k|
```
Comment (mensfeld) [inline lib/polish_geeks/dev_tools/config_manager.rb]: k is not descriptive

File: lib/polish_geeks/dev_tools/config_manager.rb (hunk 7 above)
Comment (nijikon) [inline lib/polish_geeks/dev_tools/config_manager.rb]: Nope, we want to return boolean, changing that to what you propose will end in returning nil or path.

**Pull request merged**